src/openforms/utils/form_fields.py

```diff
@@ -1,13 +1,16 @@
 from pathlib import Path
+from typing import Any, Callable, Sequence
 
+from django.contrib.postgres.fields import ArrayField
 from django.core.validators import (
     FileExtensionValidator,
     get_available_image_extensions,
 )
-from django.forms import ImageField
+from django.db import models
+from django.forms import CheckboxSelectMultiple, ImageField, MultipleChoiceField
 
 image_or_svg_extension_validator = FileExtensionValidator(
-    allowed_extensions=["svg"] + get_available_image_extensions()
+    allowed_extensions=["svg"] + list(get_available_image_extensions())
```

Comment:
The `list` here is for typing related things?

Reply:
Correct!